test/core/bad_client/bad_client.cc

```diff
@@ -128,7 +128,7 @@ void grpc_run_bad_client_test(
   GPR_ASSERT(grpc_server_has_open_connections(a.server));
 
   /* Start validator */
-  gpr_thd_new(&id, thd_func, &a, nullptr);
+  gpr_thd_new(&id, "gpr_bad_client", thd_func, &a, nullptr);
```

Comment:
in general, let's prefix these threads with `grpc_` in lieu of `gpr_`.

Reply:
Done